db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').find_in_batches do |batch|
+      batch.each do |audit|
+        audit.audited_changes.delete("password")
+        if audit.changed?
```

Comment:
you could use `.where('audited_changes like ?', '%password%')` to get just the audits that have a password change in them